### PR TITLE
add confgenerator docker support

### PR DESCRIPTION
**Pull request by eranra**

### CODE REVIEW

File: contrib/docker/Dockerfile [hunk 1]
```diff
@@ -12,16 +12,20 @@ RUN go mod download
...
-RUN apt-get update -y
-RUN apt-get install -y netbase
-RUN apt-get install -y curl
-RUN apt-get install -y net-tools
+RUN apt-get update && apt-get install -y \
+  netbase \
+  curl \
+  net-tools \
+  && rm -rf /var/lib/apt/lists/*
+
```
Comment (ronensc) [inline contrib/docker/Dockerfile:26]: 👍

File: docs/confGenerator.md [hunk 2]
```diff
@@ -30,6 +30,9 @@ Flags:
 
 ```
 
+> Note: confgenerator is available also from `netobserv/flowlogs2metrics` quay image. To use execute:
```
Comment (ronensc) [inline docs/confGenerator.md:33]: Didn't we change `flowlogs2metrics` to `flowlogs-pipeline`?

**Review by ronensc — APPROVED** [review]

File: docs/confGenerator.md (hunk 2 above)
Comment (KalmanMeth) [inline docs/confGenerator.md:33]: s/flowlogs2metrics/flowlogs-pipeline/

**Review by KalmanMeth — APPROVED** [review]

File: contrib/docker/Dockerfile (hunk 1 above)
Comment (eranra) [reply contrib/docker/Dockerfile:26]: 💯

File: docs/confGenerator.md (hunk 2 above)
Comment (eranra) [reply docs/confGenerator.md:33]: fixed

Comment (eranra) [reply docs/confGenerator.md:33]: done

**Pull request merged**